nix/quota.nix

```diff
@@ -0,0 +1,20 @@
+{ mkDerivation, base, exceptions, fetchgit, mtl, QuickCheck, stdenv
+, test-framework, test-framework-quickcheck2, transformers
+}:
+mkDerivation {
+  pname = "quota";
+  version = "0.1.0.0";
+  src = fetchgit {
```

Comment:
Missed this before, but the nixpkgs manual suggests using the most specific fetch function available, which in this case would be fetchFromGitHub.